crates/dbs-upcall/kernel/0001-dragonball-introduce-dragonball-subarch.patch

```diff
@@ -0,0 +1,854 @@
+From a3b095cbeb7d6442ac7ccfeef668dc9a7eafc9d5 Mon Sep 17 00:00:00 2001
+From: Chao Wu <chaowu@linux.alibaba.com>
+Date: Tue, 10 May 2022 17:09:40 +0800
+Subject: [PATCH 1/2] dragonball: introduce dragonball subarch
+
+Dragonball subarch is to let guest know it is in dragonball guest so
```

Comment:
subarch havs wake relationship with upcall, should we move our kernel patch to a more common folder?

Reply:
Changed.